doc/man/man3/crypto_sign_init_first_pass.3monocypher

```diff
@@ -167,17 +167,18 @@ These functions implement EdDSA with Curve25519 and Blake2b.
 This is the same as Ed25519, with Blake2b instead of SHA-512.
 Ed25519 is described in RFC 7748.
 .Sh SECURITY CONSIDERATIONS
-In addition to those documented on
-.Xr crypto_sign 3monocypher ,
-incremental signature verification encourages the user to start
-processing the message before verification is done.
+Messages are not verified until the call to
+.Xr crypto_check_final .
 Messages may be stored before they are verified, but they cannot be
 .Em trusted .
 Processing untrusted messages increases the attack surface of the
 system.
 Doing so securely is hard.
 Do not process messages before calling
 .Fn crypto_check_final .
+Additionally, the security considerations documented in
+.Xr crypto_sign 3monocypher ,
+apply here too.
```

Comment:
Actually they do not: hardware fault injection can cause a verification to fail when it should have otherwise succeeded, but the reverse is unlikely. Moreover, this can never lead to key recovery at the verification stage, because we don't process anybody's secrets here.

I think we shouldn't add this sentence.

Reply:
should it say "when signing messages, the security considerations docmented in crypto_sign also apply"?

Reply:
Oh, shit, I got confused. I thought for a moment there was a dedicated crypto_check page. Yes, "when signing messages, the security considerations docmented in crypto_sign also apply" is perfect.

Reply:
(To be clear, I retract my earlier comment: the sentence you added was fine.)